### ISSUE TITLE
No module named 'websockets.legacy' when compiling FastAPI app

### ISSUE THREAD

**Issue by sebastian-szurlej** [Nuitka/Nuitka#1547]
* Nuitka version, full Python version, flavor, OS, etc. as output by this
  command (it does more than you think, and we are adding more all the time):

0.7.7
Commercial: None
Python: 3.9.10 (tags/v3.9.10:f2f3f53, Jan 17 2022, 15:14:21) [MSC v.1929 64 bit (AMD64)]
Flavor: Unknown
Executable: D:\dev\venvs\nuitka-test\Scripts\python.exe
OS: Windows
Arch: x86_64
WindowsRelease: 10

* How did you install Nuitka and Python

pip install nuitka

* The specific PyPI names and versions

  It should be taken from this output if there specific packages involved, e.g.
  `numpy`, you are expected to shorten this to the relevant ones.

anyio==3.5.0
asgiref==3.5.0
click==8.1.3
colorama==0.4.4
fastapi==0.75.2
h11==0.13.0
httptools==0.4.0
idna==3.3
Nuitka==0.7.7
pydantic==1.9.0
python-dotenv==0.20.0
PyYAML==6.0
sniffio==1.2.0
starlette==0.17.1
typing_extensions==4.2.0
uvicorn==0.17.6
watchgod==0.8.2
websockets==10.3

Above comes from requirements.txt file, venv was clean, what was only installed was:
`pip install fastapi`
`pip install "uvicorn[standard]"`

as described here: https://fastapi.tiangolo.com/tutorial/
and also:
`pip install nuitka`

* Also supply a [Short, Self Contained, Correct, Example](https://sscce.org/)

Repro code in the link below:
https://github.com/sebastian-szurlej/nuitka-uvicorn

* Provide in your issue the Nuitka options used

nuitka --standalone main.py

After running main.exe, I'm getting error below:

Traceback (most recent call last):
  File "D:\dev\repos\nuitka-uvicorn\main.dist\main.py", line 13, in <module>
  File "D:\dev\repos\nuitka-uvicorn\main.dist\uvicorn\main.py", line 463, in run
  File "D:\dev\repos\nuitka-uvicorn\main.dist\uvicorn\server.py", line 60, in run
  File "D:\dev\repos\nuitka-uvicorn\main.dist\asyncio\runners.py", line 44, in run
  File "D:\dev\repos\nuitka-uvicorn\main.dist\asyncio\base_events.py", line 642, in run_until_complete
  File "D:\dev\repos\nuitka-uvicorn\main.dist\uvicorn\server.py", line 67, in serve
  File "D:\dev\repos\nuitka-uvicorn\main.dist\uvicorn\config.py", line 450, in load
  File "D:\dev\repos\nuitka-uvicorn\main.dist\uvicorn\importer.py", line 24, in import_from_string
ModuleNotFoundError: No module named 'websockets.legacy'

**Comment by sebastian-szurlej** [comment]
I did some testing with --include-package flag, after setting it with:
`--include-package=uvicorn`
I got the same error, however it worked with:
`--include-package=websockets`
Provided repo was a very minimal example done on my private laptop, I will test it on my desired code over the next days. I will let you know if it will also work on it.

Thanks!

**Comment by kayhayen** [comment]
Added the missing implicit dependency, should work now.

**Comment by kayhayen** [comment]
Part of 0.8 release that I just made.
